intelmq/bots/experts/http_status/expert.py

```diff
@@ -0,0 +1,59 @@
+"""
+HTTP Status Expert Bot
+
+SPDX-FileCopyrightText: 2021 Birger Schacht <schacht@cert.at>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+from typing import List
+
+from intelmq.lib.bot import Bot
+from intelmq.lib.utils import create_request_session
+
+
+class HttpStatusExpertBot(Bot):
+    """
+    Fetch the HTTP Status for a given URI
+
+    Parameters
+    ----------
+    field: str
+        The name of the field containing the URL to be checked (required).
+    success_status_codes: List
+        A list of success status codes. If this parameter is omitted or the list is empty,
+        successful status codes are the ones between 200 and 400.
+    """
+    field: str = None  # The field containing the URI
```

Comment:
Isn't it better if this field would be `url` instead of `field` as its a httpStatusExpert?

Reply:
Oh, now I got it. Was a bit confused. Thanks for clarification.